internal/service/resource_pool_object_handler.go

```diff
@@ -160,7 +160,7 @@ func (b *ResourcePoolObjectHandlerBuilder) Build() (
 
 // Get is part of the implementation of the collection handler interface.
 func (h *ResourcePoolObjectHandler) Get(ctx context.Context,
```

Comment:
Can we remove also this file now? I.e. use a single handler in start_resource_server

Reply:
Yes, I think so. Let me take a look at that after merging this.